qemu/qmp.go

```diff
@@ -267,10 +267,7 @@ func (q *QMP) readLoop(fromVMCh chan<- []byte) {
 
 	for scanner.Scan() {
 		line := scanner.Bytes()
-		if q.cfg.Logger.V(1) {
-			q.cfg.Logger.Infof("read from QMP: %s", string(line))
```

Comment:
Why not just set it to debug or trace?

Reply:
Let me try w/ trace.

Reply:
There aren't semantics built in for trace, and if I wrap the print with q.cfg.Logger.V(7), I still see the output on kata side.

Reply:
Okay, okay.
I'll add govmm as one part that has to also be addressed together with https://github.com/kata-containers/kata-containers/issues/1600